src/main/kotlin/app/revanced/cli/command/MainCommand.kt

```diff
@@ -158,11 +159,15 @@ internal object MainCommand : Runnable {
         val adb: Adb? = _args.deploy?.let {
             Adb(outputFile, patcher.data.packageMetadata.packageName, _args.deploy!!, !args.mount)
         }
-        val patchedFile = if (args.mount) outputFile
-        else File(args.cacheDirectory).resolve("${outputFile.nameWithoutExtension}_raw.apk")
+        val patchedFile = File(args.cacheDirectory).resolve("${outputFile.nameWithoutExtension}_raw.apk")
 
         Patcher.start(patcher, patchedFile)
 
+        // align the inputFile and write to alignedOutput
```

Comment:
Please create a class for this too the same way the signer has.

Reply:
Sure.

Reply:
ZipAligner already has a class so i'll modify that.

Reply:
Done.